src/editor/handgizmo.cpp

```diff
@@ -0,0 +1,78 @@
+#include "handgizmo.h"
+
+class HandGizmoMaterial : public iris::Material
+{
+public:
+	QColor color;
+	float fresnelPow;
+
+	HandGizmoMaterial()
+	{
+		auto shader = iris::Shader::load(
+			":assets/shaders/default_material.vert",
+			IrisUtils::getAbsoluteAssetPath("app/shaders/fresnel.frag")
+			);
+		this->setShader(shader);
+
+		this->renderLayer = (int)iris::RenderLayer::Transparent;
+		this->renderStates.blendState = iris::BlendState::createAlphaBlend();
+		//this->renderStates.rasterState.depthBias = 1;// this ensures it's always on top
+		//this->renderStates.rasterState.depthScaleBias = 1;// this ensures it's always on top
+
+		color = QColor(41, 128, 185);
+		fresnelPow = 2;
+
+		this->enableFlag("SKINNING_ENABLED");
+	}
+
+	void begin(iris::GraphicsDevicePtr device, iris::ScenePtr scene) override
+	{
+		iris::Material::begin(device, scene);
+
+		device->setShaderUniform("u_color", color);
+		device->setShaderUniform("u_fresnelPow", fresnelPow);
+	}
+};
+
+HandGizmoHandler::HandGizmoHandler()
+{
+	handScale.setToIdentity();
+	handScale.scale(0.01f);// scale by 1/100;
```

Comment:
Is this to accomodate the c4d exporter? Can't we use collada instead or maybe try experimental gltf? Very bizarre.

Reply:
The scale of the model coming the oculus website is too large